open-metadata-test/open-metadata-fvt/access-services-fvt/subject-area-fvt/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/fvt/RelationshipsFVT.java

```diff
@@ -198,6 +199,9 @@ public void run() throws SubjectAreaFVTCheckedException, InvalidParameterExcepti
         projectFVT.deleteProject(project.getSystemAttributes().getGUID());
         projectFVT.purgeProject(project.getSystemAttributes().getGUID());
 
+        Category cat3 = catFVT.createCategory(DEFAULT_TEST_CAT_NAME3, glossaryGuid);
+        Category cat4 = catFVT.createCategory(DEFAULT_TEST_CAT_NAME4, glossaryGuid);
+        categoryHierarchyLinkFVT(cat3, cat4);
     }
```

Comment:
Could we get cat3 and validate that it has a parent of cat4 as well please.

Reply:
FVTUtils.checkGuidEnd1s("CategoryHierarchyLink", parent.getSystemAttributes().getGUID(), categoryHierarchyLink.getSuperCategoryGuid());
        FVTUtils.checkGuidEnd2s("CategoryHierarchyLink", child.getSystemAttributes().getGUID(), categoryHierarchyLink.getSubCategoryGuid());


Its check on 1456 and 1457 lines